mne/viz/_3d.py

```diff
@@ -481,6 +481,10 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
         `'$SUBJECTS_DIR/$SUBJECT/bem/$SUBJECT-$SOURCE.fif'`, and then look for
         `'$SUBJECT*$SOURCE.fif'` in the same directory. For `'outer_skin'`,
         the subjects bem and bem/flash folders are searched. Defaults to None.
+    fig : mayavi figure object | None
+        Mayavi Scene (instance of mlab.Figure) in which to plot the alignment.
+        If ``None``, creates a new 600x600 pixel figure with black background.
+        .. versionadded:: 0.16
```

Comment:
It needs a preceding newline or [it won't render properly](https://6322-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.xrDTukx/html/generated/mne.viz.plot_alignment.html#mne.viz.plot_alignment)